brion/morphologies.h

```diff
@@ -0,0 +1,115 @@
+/* Copyright (c) 2013-2015, EPFL/Blue Brain Project
+ *                          Juan Hernando <jhernando@fi.upm.es>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef BRION_MORPHOLOGIES
+#define BRION_MORPHOLOGIES
+
+/**
+ * @file
+ * Free functions for loading sets of morphologies
+ */
+
+#include <brion/api.h>
+#include <brion/types.h>
+
+namespace brion
+{
+
+/** @namespace brion::morphologies
+ *  This namespace contains several free functions for loading sets of
+ *  morphologies.
```

Comment:
This convenience of loading is almost the same we have with the SDK today. Together with its shortcomings. In-core and no lazy-loading. Do we really gain much by providing those? I agree on having some convenience to construct the paths/URIs for loading from a folder for instance, but the actual loading should be done by the application/user-code right now.